libs/dh/wholesale/feature-calculations/src/lib/details/details.component.html

```diff
@@ -56,28 +56,31 @@
   }
   <watt-drawer-heading>
     <div class="headline-container">
-      <h3
-        [wattCopyToClipboard]="id()"
-        [wattTooltip]="t('details.copyId')"
-        wattTooltipPosition="right"
-        class="headline watt-headline-2"
-      >
+      <h3 class="headline watt-headline-2">
         {{ t("calculationTypes." + (calculation?.calculationType ?? "UNKNOWN")) }}
       </h3>
     </div>
-    <watt-description-list>
+    <watt-description-list [groupsPerRow]="3">
       <watt-description-list-item
         [label]="t('details.startedBy')"
         [value]="calculation?.createdByUserName | dhEmDashFallback"
       />
       <watt-description-list-item
-        [label]="t('details.period')"
-        [value]="calculation?.period | wattDate: 'short' | dhEmDashFallback"
+        [label]="t('details.executionType')"
+        [value]="calculation && t('executionTypes.' + calculation.executionType) | dhEmDashFallback"
```

Comment:
Question: && translation will ever be null or undefind, so dhEMdashFallback is never used?